cpu/rpx0xx/Makefile.include

```diff
@@ -14,7 +14,7 @@ INCLUDES += -I$(RIOTCPU)/rpx0xx/include
 VECTORS_O       ?= $(BINDIR)/cpu/vectors.o
 VECTORS_FILE    := $(RIOTCPU)/rpx0xx/vectors.c
 
-PROGRAMMERS_SUPPORTED   := uf2conv openocd jlink
+PROGRAMMERS_SUPPORTED   := uf2conv openocd jlink elf2uf2
```

Comment:
Since uf2conv is not working, shouldn't it be dropped? I think `elf2uf2` could also be the first entry in the list. I have to admit that `uf2conv` is not working with Alpine anyway, so I always went the openocd rout anyway. But I guess most users will mostly use the UF2 path for programming, so it makes IMO sense to have this first.

Reply:
I agry with all comments. Moreover, I added elf2uf2 as a default PROGRAMMER in Makefile.include. All commits squashed.